### DISCUSSION TITLE
General question about the project

### DISCUSSION THREAD

**Discussion by AlexandreEichenberger** [ENOT-AutoDL/onnx2torch#200] [General]
I am involved with ONNX and would like to know a bit more about the overall goals of this project.  

My initial 2 questions are:

1. Are your plan to update to the latest ONNX releases? I read that you are currently at opset 16, the current ONNX release is at 20. A recent development that might be interesting is that some ONNX operations are now defined as functions of simpler operators, so possibly exploiting that feature may provide a wider coverage.
2. How do you decide which version of PyTorch to target? Is it a fixed target, or does it evolve over time?

Looking forward to hearing back from you.

**Comment by senysenyseny16** [comment]
Hi @AlexandreEichenberger ,

Thank you for your interest in the project.

> Are your plan to update to the latest ONNX releases?
> I read that you are currently at opset 16, the current ONNX release is at 20.

Yes, we plan to support 20 opset, but at the moment we do not have the resources.

> A recent development that might be interesting is that some ONNX operations
> are now defined as functions of simpler operators, so possibly exploiting that feature may provide a wider coverage.

It would be interesting to see an example.

> How do you decide which version of PyTorch to target? Is it a fixed target, or does it evolve over time?

We started developing this project with PyTorch 1.8 and tested it on every version up to PyTorch 2.1.
